publication-commons/src/main/java/no/unit/nva/publication/permission/strategy/PermissionStrategy.java

```diff
@@ -45,6 +46,17 @@ protected boolean isDegree() {
                    .orElse(false);
     }
 
+    protected boolean isEmbargoDegree() {
+        return publication.getAssociatedArtifacts().stream()
```

Comment:
Have to add a check for if the publication is a degree or not. A "normal" publication might also have embargo.

Reply:
```suggestion
        return isDegree() && publication.getAssociatedArtifacts().stream()
```

Reply:
Think this should do the trick?